defaults/main.yml

```diff
@@ -138,6 +138,8 @@ postgresql_ssl_ciphers:
   - "@STRENGTH"
 postgresql_ssl_prefer_server_ciphers: on
 postgresql_ssl_ecdh_curve: "prime256v1"
+postgresql_ssl_min_protocol_version: "TLSv1"                        # (>= 12)
+postgresql_ssl_max_protocol_version ""                              # (>= 12)
```

Comment:
@gclough is this a typo?

Reply:
Yes, sorry... first attempt.  I'll get it working and send it to you for a review.

Reply:
Don't be sorry! I was just glancing over the PR quickly. Thanks for tackling this!

Reply:
It seems that Ubuntu 14 isn't supported any longer... so I've removed that and added 18.

Reply:
See later commits, and review if you can please.